MUI_TimerBars/MUI_TimerBars.lua

```diff
@@ -678,6 +679,11 @@ do
 
                     ---@param bar TimerBar
                     for i, bar in ipairs(data.activeBars) do
+						auraInfo = GetAuraInfoByAuraID(self.field:GetUnitID(), bar.auraId, bar.AuraType);
```

Comment:
Also, I think it is "bar.AuraId" because public properties have a capital and other references seem to be capitals.